queries/lua/injections.scm

```diff
@@ -135,6 +135,12 @@
   (#set! injection.language "luadoc")
   (#offset! @injection.content 0 1 0 0))
 
+(comment
+  content: (_) @injection.content
+  (#lua-match? @injection.content "^[-][|]")
```

Comment:
Why not combine with the previous pattern?
```
(#lua-match? @injection.content "^[-][%s]*[|]")
```

Reply:
I was debating that but my rationale was that it might let in more false positives - but I did check lua-ls and spaces don't seem to matter, my bad